services/QuillLMS/lib/tasks/flags.rake

```diff
@@ -16,6 +16,18 @@ namespace :flags do
   end
 
   namespace :users do
+    desc 'add custom flagsets for quill.org emails'
+    task :add_custom_flagsets => :environment do
+      college_board_ids = User.find_by_sql(
```

Comment:
Minor nitpick, but `college_board_ids` is probably not a useful name for this value.  Probably left over from copy/paste.